app/controllers/admin/tags_controller.rb

```diff
@@ -78,11 +74,14 @@ def filtered_tags
       scope = scope.unreviewed if filter_params[:review] == 'unreviewed'
       scope = scope.reviewed.order(reviewed_at: :desc) if filter_params[:review] == 'reviewed'
       scope = scope.pending_review.order(requested_review_at: :desc) if filter_params[:review] == 'pending_review'
-      scope.order(max_score: :desc)
+      scope = scope.matches_name(filter_params[:name].to_s.strip) unless filter_params[:name].nil?
```

Comment:
Are there security concerns about the entered value?

Reply:
Seems fine to me.